### DISCUSSION TITLE
Can anyone advise how to pair DFC w/ SS2K?

### DISCUSSION THREAD

**Discussion by StrangeMagic1** [doudar/SmartSpin2k#375] [Q&A] [answered]
Hi,
I just got a prebuilt SS2K because I am not tech savvy at all.  I have a Peloton and a Data Fitness Connector.  When I scan Bluetooth, I don't see DFC show up as an option. I only see any and none.  No sign of my DFC.  How do I pair my DFC w/ the SS2K?

Also does anyone have a diagram of this set up on a Peloton?  I'm also struggling with how the pieces fit together....  Thanks.

**Comment by doudar** [comment]
Are you able to get a log while you scan or try to connect to the DFC using these instructions?

https://github.com/doudar/SmartSpin2k/wiki/Viewing-logs-via-UDP

Also, you’ll need to make sure the DFC isn’t connected to anything else before you connect it. It might help to power cycle the DFC immediately before trying.

Thank you!

**Comment by eMadman** [comment, marked as answer]
I think I helped you over email the other day!  

Regardless, these steps will help get it going:

- make sure no other apps or devices  are connected to the DFC.  Bluetooth bike sensors can only be connected to one device at a time.  We need DFC to be available for ss2k.
- press and hold both buttons on the shifter for 3 seconds (a bit longer is fine).  This will force the ss2k to reconnect to all Bluetooth devices and it ought to pick up the DFC when this happens (takes up to 30 seconds). This will also pick up your heart rate monitor If you have it turned on.

If the above doesn't work, you may need to restart the DFC and do the two buttons for 3 seconds thing again if it doesn't work the first time

This should get you going.  There's another way to do this through the smartspin2k web interface but let's try this first since it's the fastest option
